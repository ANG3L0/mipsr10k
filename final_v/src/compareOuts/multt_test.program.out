Chronologic VCS simulator copyright 1991-2011
Contains Synopsys proprietary information.
Compiler version E-2011.03_Full64; Runtime version E-2011.03_Full64;  Dec 10 00:51 2012
@@
@@
@@                     0  Asserting System reset......
@@
@@                     0 : System STILL at reset, can't show anything
@@
@@
@@                    10 : System STILL at reset, can't show anything
@@
@@
@@                    20 : System STILL at reset, can't show anything
@@
@@@ Unified Memory contents hex on left, decimal on right: 
@@@
@@@ mem[    0] = 205f000a203f0001 : 2332583175491878913
@@@ mem[    8] = 209f0008207f0002 : 2350597565415620610
@@@ mem[   16] = 4c6304034c630403 : 5504247581789848579
@@@ mem[   24] = 4c6304034c630403 : 5504247581789848579
@@@ mem[   32] = 4c6304034c630403 : 5504247581789848579
@@@ mem[   40] = 4c6304034c630403 : 5504247581789848579
@@@ mem[   48] = 4c6304034c630403 : 5504247581789848579
@@@ mem[   56] = 4c6304034c630403 : 5504247581789848579
@@@ mem[   64] = 4c6304034c630403 : 5504247581789848579
@@@ mem[   72] = 4c6304034c630403 : 5504247581789848579
@@@ mem[   80] = 4c6304034c630403 : 5504247581789848579
@@@ mem[   88] = 4c6304034c630403 : 5504247581789848579
@@@ mem[   96] = 4c6304034c630403 : 5504247581789848579
@@@ mem[  104] = 4c6304034c630403 : 5504247581789848579
@@@ mem[  112] = 4c6304034c630403 : 5504247581789848579
@@@ mem[  120] = 4c6304034c630403 : 5504247581789848579
@@@ mem[  128] = 4c6304034c630403 : 5504247581789848579
@@@ mem[  136] = 4c6304034c630403 : 5504247581789848579
@@@ mem[  144] = 4c6304034c630403 : 5504247581789848579
@@@ mem[  152] = 4c6304034c630403 : 5504247581789848579
@@@ mem[  160] = 4c6304034c630403 : 5504247581789848579
@@@ mem[  168] = 0000000000000555 : 1365
@@@
@@@
@@                  3542 : System halted
@@
@@@ System halted on HALT instruction
@@@
@@
@@  353 cycles / 43 instrs = 8.209302 CPI
@@
@@  10560.00 ns total time to execute
@@

$finish called from file "testbench/pipeline_test.v", line 817.
$finish at simulation time                 3542
           V C S   S i m u l a t i o n   R e p o r t 
Time: 3542
CPU Time:      0.330 seconds;       Data structure size:   0.3Mb
Mon Dec 10 00:51:23 2012
