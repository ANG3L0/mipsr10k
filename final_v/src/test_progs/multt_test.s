
        data1 = 0x0001   
        data2 = 0x000a
        data3 = 0x0002
        data4 = 0x0008

        lda  $r1, data1   /* $r1 = 1 */
        lda  $r2, data2   /* $r2 = 2 */
        lda  $r3, data3   /* $r3 = 6 */
        lda  $r4, data4   /* $r4 = 8 */
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
#       mult $r, $r1, $r2
        call_pal 0x555
