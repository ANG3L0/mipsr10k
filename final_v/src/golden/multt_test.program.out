Chronologic VCS simulator copyright 1991-2012
Contains Synopsys proprietary information.
Compiler version G-2012.09_Full64; Runtime version G-2012.09_Full64;  Dec 10 00:19 2012
@@
@@
@@                     0  Asserting System reset......
@@
@@                     0 : System STILL at reset, can't show anything
@@
@@
@@                   100 : System STILL at reset, can't show anything
@@
@@
@@                   200 : System STILL at reset, can't show anything
@@
@@
@@                   300 : System STILL at reset, can't show anything
@@
@@                   360  Deasserting System reset......
@@
@@
@@@ Unified Memory contents hex on left, decimal on right: 
@@@
@@@ mem[    0] = 205f000a203f0001 : 2332583175491878913
@@@ mem[    8] = 209f0008207f0002 : 2350597565415620610
@@@ mem[   16] = 4c6304034c630403 : 5504247581789848579
@@@ mem[   24] = 4c6304034c630403 : 5504247581789848579
@@@ mem[   32] = 4c6304034c630403 : 5504247581789848579
@@@ mem[   40] = 4c6304034c630403 : 5504247581789848579
@@@ mem[   48] = 4c6304034c630403 : 5504247581789848579
@@@ mem[   56] = 4c6304034c630403 : 5504247581789848579
@@@ mem[   64] = 4c6304034c630403 : 5504247581789848579
@@@ mem[   72] = 4c6304034c630403 : 5504247581789848579
@@@ mem[   80] = 4c6304034c630403 : 5504247581789848579
@@@ mem[   88] = 4c6304034c630403 : 5504247581789848579
@@@ mem[   96] = 4c6304034c630403 : 5504247581789848579
@@@ mem[  104] = 4c6304034c630403 : 5504247581789848579
@@@ mem[  112] = 4c6304034c630403 : 5504247581789848579
@@@ mem[  120] = 4c6304034c630403 : 5504247581789848579
@@@ mem[  128] = 4c6304034c630403 : 5504247581789848579
@@@ mem[  136] = 4c6304034c630403 : 5504247581789848579
@@@ mem[  144] = 4c6304034c630403 : 5504247581789848579
@@@ mem[  152] = 4c6304034c630403 : 5504247581789848579
@@@ mem[  160] = 4c6304034c630403 : 5504247581789848579
@@@ mem[  168] = 0000000000000555 : 1365
@@@
@@@
@@                 26620 : System halted
@@
@@@ System halted on HALT instruction
@@@
@@
@@  263 cycles / 42 instrs = 6.261905 CPI
@@
@@  7860.00 ns total time to execute
@@

$finish called from file "testbench/testbench.v", line 290.
$finish at simulation time                27620
           V C S   S i m u l a t i o n   R e p o r t 
Time: 2762000 ps
CPU Time:      0.290 seconds;       Data structure size:   0.1Mb
Mon Dec 10 00:19:35 2012
